Std/Data/Nat/Lemmas.lean

```diff
@@ -971,3 +1117,17 @@ protected theorem dvd_of_mul_dvd_mul_right (kpos : 0 < k) (H : m * k ∣ n * k)
 
 @[simp] theorem sum_append : Nat.sum (l₁ ++ l₂) = Nat.sum l₁ + Nat.sum l₂ := by
   induction l₁ <;> simp [*, Nat.add_assoc]
+
+/-! ## deprecated -/
+
+@[deprecated Nat.max_zero_left]
```

Comment:
Why the rename? I don't see any rationale given,  and TBH I prefer the original name.